### PR TITLE
HDDS-1971. Update document for HDDS-1891: Ozone fs shell command should work with default port when port number is not specified

### PR DESCRIPTION
**Pull request by smengcl**

### CODE REVIEW

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 1]
```diff
@@ -77,13 +77,39 @@ Or put command etc. In other words, all programs like Hive, Spark, and Distcp wi
...
+hdfs dfs -ls o3fs://bucket.volume.om-host.example.com/key
+{{< /highlight >}}
+
+The above command is essentially equivalent to:
+
+{{< highlight bash>}}
+hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:6789/key
+{{< /highlight >}}
+
+Note only the port number in the config is relevant. The host name in config `ozone.om.address` is ignored in this case.
```
Comment (bharatviswa504) [inline hadoop-hdds/docs/content/interface/OzoneFS.md]: Can we reword this as:
Note: In the above case, port number from the config is used, whereas hostname in the config `ozone.om.address` is ignored.

File: hadoop-hdds/docs/content/interface/OzoneFS.md [hunk 2]
```diff
@@ -77,13 +77,39 @@ Or put command etc. In other words, all programs like Hive, Spark, and Distcp wi
...
-Moreover, the filesystem URI can take a fully qualified form with the OM host and port as a part of the path following the volume name.
-For example,
+Moreover, the filesystem URI can take a fully qualified form with the OM host and an optional port as a part of the path following the volume name.
+For example, you can specify both host and port:
 
 {{< highlight bash>}}
 hdfs dfs -ls o3fs://bucket.volume.om-host.example.com:5678/key
 {{< /highlight >}}
 
+When the port number is not specified, it will be retrieved from config key `ozone.om.address`.
```
Comment (bharatviswa504) [inline hadoop-hdds/docs/content/interface/OzoneFS.md]: When the port number is not specified, it will be retrieved from config key `ozone.om.address` if defined or fall back to default port 9862.

With this, we can eliminate, last lines from 109-111.

**Comment by hadoop-yetus** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 126 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 799 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 1753 | branch has no errors when building and testing our client artifacts. |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 624 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 736 | patch has no errors when building and testing our client artifacts. |
||| _ Other Tests _ |
| +1 | asflicense | 51 | The patch does not generate ASF License warnings. |
| | | 3457 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.0 Server=19.03.0 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1306/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1306 |
| Optional Tests | dupname asflicense mvnsite |
| uname | Linux 1bc910d40dad 4.15.0-52-generic #56-Ubuntu SMP Tue Jun 4 22:49:08 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 9b8359b |
| Max. process+thread count | 296 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/docs U: hadoop-hdds/docs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1306/1/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/docs/content/interface/OzoneFS.md (hunk 1 above)
Comment (smengcl) [reply hadoop-hdds/docs/content/interface/OzoneFS.md]: done

File: hadoop-hdds/docs/content/interface/OzoneFS.md (hunk 2 above)
Comment (smengcl) [reply hadoop-hdds/docs/content/interface/OzoneFS.md]: done. still keeping L109 but repharsed.

**Comment by bharatviswa504** [comment]
Thank You @smengcl for the contribution.
I will commit this to the trunk.
